curve25519-dalek/src/backend/serial/u64/scalar.rs

```diff
@@ -814,15 +816,31 @@ impl Scalar52 {
             limbs_bounded(self),
         ensures
             limbs_bounded(&result),
-            to_nat(&result.limbs) % group_order() == (to_nat(&self.limbs) * montgomery_radix())
-                % group_order(),
+            #[trigger] (to_nat(&result.limbs) % group_order()) == 
+                #[trigger] ((to_nat(&self.limbs) * montgomery_radix()) % group_order()),
```

Comment:
If it's not a `broadcast` fn or a `forall`, you don't need to add `#[trigger]` (https://verus-lang.github.io/verus/guide/trigger-annotations.html#trigger-annotations)

Reply:
```suggestion
            to_nat(&result.limbs) % group_order() == (to_nat(&self.limbs) * montgomery_radix())
                % group_order(),
```